package.json

```diff
@@ -15,14 +15,13 @@
     "prepack": "mv src/package.json src_package.json",
     "postpack": "mv src_package.json src/package.json",
     "start": "webpack-dev-server --config configs/webpack.dev.config.js",
-    "start:https": "webpack-dev-server --config configs/webpack.dev.config.js --https --pfx configs/webpack.docs.dev.pfx",
-    "serve:e2e": "lite-server -c=e2e/bs-config.e2e.json",
-    "e2e": "concurrently \"npm run serve:e2e\" \"grunt protractor:e2e\" --kill-others --success first",
-    "build:library": "ng-packagr -p ./src/ng-package.json",
-    "build:documentation:prod": "node --max-old-space-size=4096 ./node_modules/webpack/bin/webpack.js --colors --config ./configs/webpack.docs.prod.config.js",
-    "package:library": "grunt build:library",
-    "lint": "grunt lint",
-    "e2e:develop": "concurrently \"webpack --watch --progress --colors --config configs/webpack.e2e.config.js\" \"tsc -w --project .\\e2e\\tsconfig.json\" \"npm run serve:e2e\""
+    "start-https": "npm run start -- --https --pfx configs/webpack.docs.dev.pfx",
+    "e2e": "concurrently \"npm run e2e-serve\" \"grunt protractor:e2e\" --kill-others --success first",
```

Comment:
As I recall, `npm run e2e` does not work for running the e2e tests. So this script should just call `grunt e2e`. Something like https://github.com/sindresorhus/grunt-concurrent might help to move this out of package.json and also remove the need for this global tool.

Comparing with the package.json for Material, perhaps "test:e2e" would be a better name here. Then we can have something like `"test": "grunt test && grunt e2e"` for a full test run (which would be the nightly build).